--- debootstrap.orig	2013-04-04 18:58:36.000000000 +0400
+++ debootstrap	2013-05-19 23:57:50.596593154 +0400
@@ -17,7 +17,7 @@
 	fi
 fi
 
-DEVICES_TARGZ=$DEBOOTSTRAP_DIR/devices.tar.gz
+DEVICES_TARGZ=$DEBOOTSTRAP_DIR/devices.tgz
 
 . $DEBOOTSTRAP_DIR/functions
 exec 4>&1
@@ -619,7 +619,8 @@
 	if ! am_doing_phase second_stage; then
 		cp "$0"				 "$TARGET/debootstrap/debootstrap"
 		cp $DEBOOTSTRAP_DIR/functions	 "$TARGET/debootstrap/functions"
-		cp $DEBOOTSTRAP_DIR/devices.tar.gz	 "$TARGET/debootstrap/devices.tar.gz"
+		cp $DEBOOTSTRAP_DIR/devices.tgz	 "$TARGET/debootstrap/devices.tgz"
 		cp $SCRIPT			 "$TARGET/debootstrap/suite-script"
 		echo "$ARCH"			>"$TARGET/debootstrap/arch"
 		echo "$SUITE"			>"$TARGET/debootstrap/suite"
