--- debootstrap.orig	2015-10-02 23:14:11.180424159 +0300
+++ debootstrap	2015-10-02 23:12:13.630918463 +0300
@@ -17,7 +17,7 @@
 	fi
 fi
 
-DEVICES_TARGZ=$DEBOOTSTRAP_DIR/devices.tar.gz
+DEVICES_TARGZ=$DEBOOTSTRAP_DIR/devices.tgz
 
 . $DEBOOTSTRAP_DIR/functions
 exec 4>&1
@@ -617,9 +617,9 @@
 	first_stage_install
 
 	if ! am_doing_phase second_stage; then
-		cp "$0"				 "$TARGET/debootstrap/debootstrap"
+		cp $DEBOOTSTRAP_DIR/debootstrap	 "$TARGET/debootstrap/debootstrap"
 		cp $DEBOOTSTRAP_DIR/functions	 "$TARGET/debootstrap/functions"
-		cp $DEBOOTSTRAP_DIR/devices.tar.gz	 "$TARGET/debootstrap/devices.tar.gz"
+		cp $DEBOOTSTRAP_DIR/devices.tgz	 "$TARGET/debootstrap/devices.tgz"
 		cp $SCRIPT			 "$TARGET/debootstrap/suite-script"
 		echo "$ARCH"			>"$TARGET/debootstrap/arch"
 		echo "$SUITE"			>"$TARGET/debootstrap/suite"
